src/Codata/Sized/Cofin/Literals.agda

```diff
@@ -8,12 +8,12 @@
 
 module Codata.Sized.Cofin.Literals where
 
-open import Data.Nat.Base
-open import Agda.Builtin.FromNat
-open import Codata.Sized.Conat
-open import Codata.Sized.Conat.Properties
-open import Codata.Sized.Cofin
-open import Relation.Nullary.Decidable
+open import Data.Nat.Base using (ℕ; suc)
+open import Agda.Builtin.FromNat using (Number)
+open import Codata.Sized.Conat using (Conat)
+open import Codata.Sized.Conat.Properties using (_ℕ≤?_)
+open import Codata.Sized.Cofin using (Cofin; fromℕ<)
+open import Relation.Nullary.Decidable using (True; toWitness)
```

Comment:
`style-guide` mandates *alphabetical* order for `import`s:
```suggestion
open import Agda.Builtin.FromNat using (Number)
open import Codata.Sized.Conat using (Conat)
open import Codata.Sized.Conat.Properties using (_ℕ≤?_)
open import Codata.Sized.Cofin using (Cofin; fromℕ<)
open import Data.Nat.Base using (ℕ; suc)
open import Relation.Nullary.Decidable.Core using (True; toWitness)
```

NB moreover the last one can be narrowed to `Core`.

Reply:
However, I do make an exception for the rest of the modules touched here, in that `Level` and `Size` can be imported ahead of anything else.